hedera-node/src/test/java/com/hedera/services/contracts/execution/CallLocalEvmTxProcessorTest.java

```diff
@@ -97,7 +97,8 @@ private void setup() {
                         operations,
                         precompiledContractMap,
                         aliasManager,
-                        storageExpiry);
+                        storageExpiry,
+                        new FakeHederaTracer());
```

Comment:
Are there any asserts we should have on this take?

Reply:
As mentioned above, it's been removed now